x-pack/plugins/observability_solution/logs_shared/public/components/logging/log_text_stream/highlighting.tsx

```diff
@@ -7,33 +7,18 @@
 
 import React from 'react';
 
-import { euiStyled } from '@kbn/kibana-react-plugin/common';
-import { chooseLightOrDarkColor, tintOrShade } from '../../../utils/styles';
+import styled from '@emotion/styled';
 
-export const ActiveHighlightMarker = euiStyled.mark`
-  color: ${(props) =>
-    chooseLightOrDarkColor(
-      props.theme.eui.euiColorAccent,
-      props.theme.eui.euiColorEmptyShade,
-      props.theme.eui.euiColorDarkestShade
-    )};
-  background-color: ${(props) => props.theme.eui.euiColorAccent};
-  outline: 1px solid ${(props) => props.theme.eui.euiColorAccent};
-  };
+export const ActiveHighlightMarker = styled.mark`
+  color: ${({ theme }) => theme.euiTheme.colors.textParagraph};
+  background-color: ${({ theme }) => theme.euiTheme.colors.backgroundFilledAccent};
+  outline: 1px solid ${({ theme }) => theme.euiTheme.colors.backgroundFilledAccent};
```

Comment:
Suggestion: We can use `euiTheme.border.width.thin` here as outline width. (same for `HighlightMarker`)

Reply:
Addressed with [style(logs): update style suggestions](https://github.com/elastic/kibana/pull/202746/commits/7b12855cc171332cadbb533ee98505855aa016b6)